### ISSUE TITLE
[ISSUE] crc-tray.exe detected as malicious process

### ISSUE THREAD

**Issue by Tal-Hason** [code-ready/crc#3114]
### General information

  * OS: Windows
  * Hypervisor: Hyper-V 
  * Did you run `crc setup` before starting it Yes
  * Running CRC on: Laptop 

## CRC version 
```bash
# Put `crc version` output here
>>> crc version
CodeReady Containers version: 2.0.1+f0fcd50e
OpenShift version: 4.10.3
Podman version: 3.4.4
```
  
## CRC status
```bash
# Put `crc status --log-level debug` output here
>>> crc status --log-level debug
DEBU CodeReady Containers version: 2.0.1+f0fcd50e
DEBU OpenShift version: 4.10.3
DEBU Podman version: 3.4.4
DEBU Running 'crc status'
DEBU Checking file: C:\Users\thason\.crc\machines\crc\.crc-exist
DEBU Checking file: C:\Users\thason\.crc\machines\crc\.crc-exist
DEBU Running 'Hyper-V\Get-VM crc | Select-Object -ExpandProperty State'
CRC VM:          Stopped
OpenShift:       Stopped (v4.10.3)
Podman:
Disk Usage:      0B of 0B (Inside the CRC VM)
Cache Usage:     18.16GB
Cache Directory: C:\Users\thason\.crc\cache
DEBU Running '(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion" -Name ReleaseId).ReleaseId'
DEBU Running '(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").EditionID'

```

## CRC config
```bash
# Put `crc config view` output here
>>>crc config view
- consent-telemetry                     : yes
- cpus                                  : 6
- memory                                : 32678
```

## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows
>>> systeminfo

Host Name:                ****************
OS Name:                   Microsoft Windows 11 Enterprise
OS Version:                10.0.22000 N/A Build 22000
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Member Workstation
OS Build Type:             Multiprocessor Free
Registered Owner:          Red Hat User
Registered Organization:   Red Hat Inc.
Product ID:                00329-00000-00003-AA136
Original Install Date:     03/08/2022, 12:33:34
System Boot Time:          04/14/2022, 08:05:58
System Manufacturer:       LENOVO
System Model:              20TJS2F40Y
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 165 Stepping 2 GenuineIntel ~2310 Mhz
BIOS Version:              LENOVO N2VET38W (1.23 ), 02/10/2022
Windows Directory:         C:\WINDOWS
System Directory:          C:\WINDOWS\system32
Boot Device:               \Device\HarddiskVolume4
System Locale:             he;Hebrew
Input Locale:              en-us;English (United States)
Time Zone:                 (UTC+02:00) Jerusalem
Total Physical Memory:     65,289 MB
Available Physical Memory: 42,222 MB
Virtual Memory: Max Size:  75,017 MB
Virtual Memory: Available: 46,386 MB
Virtual Memory: In Use:    28,631 MB
Page File Location(s):     C:\pagefile.sys
Domain:                    win.redhat.com
Logon Server:              \\AMS2-DC02
Hotfix(s):                 3 Hotfix(s) Installed.
                           [01]: KB5007575
                           [02]: KB5010386
                           [03]: KB5009641
Network Card(s):           4 NIC(s) Installed.
                           [01]: Viscosity Virtual TUN Adapter
                                 Connection Name: Tel Aviv (TLV)
                                 Status:          Hardware not present
                           [02]: Intel(R) Wi-Fi 6 AX201 160MHz
                                 Connection Name: Wi-Fi
                                 DHCP Enabled:    Yes
                                 DHCP Server:     192.168.0.1
                                 IP address(es)
                                 [01]: 192.168.0.104
                                 [02]: fe80::d11b:5a94:fd02:ca5e
                           [03]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (crc)
                                 DHCP Enabled:    Yes
                                 DHCP Server:     255.255.255.255
                                 IP address(es)
                                 [01]: 169.254.109.134
                                 [02]: fe80::800d:e8a2:1f7d:6d86
                           [04]: Bluetooth Device (Personal Area Network)
                                 Connection Name: Bluetooth Network Connection
                                 Status:          Media disconnected
Hyper-V Requirements:      A hypervisor has been detected. Features required for Hyper-V will not be displayed.
```

### Steps to reproduce

  1.  Install CRC latest version(2.0.1+f0fcd50e) on a PC with Antivirus (mine has SentinalOne)
  2.  After Installation the SentianlOne Quarantine the "crc-tray.exe" file and Shortcut
  3. 
  4. 

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.
![Screenshot 2022-04-13 135923](https://user-images.githubusercontent.com/97454240/163785804-abfe3d82-a832-48f2-8f4c-154f0249d667.png)
![Screenshot 2022-04-13 154735](https://user-images.githubusercontent.com/97454240/163785826-633fc057-e6f2-4d87-a369-5cd29ea504e7.png)

**Comment by anjannath** [comment]
With the latest release `2.0.1` this file should not be created if you are using the `msi` for installing, did you have an older release of `CRC` installed?

This is a false positive, we have seen this earlier, we create the `CodeReady Containers.lnk` file to make the tray app auto-start, but with the latest release we are using a different method to auto-start the tray and removed the code that was creating this shortcut. (https://github.com/code-ready/crc/commit/94bb587f616f07049ec7f580fd3d2ff8117a0b59)

So if you upgrade from an older release using the msi the shortcut should've been removed, if not we have bug in the msi.

**Comment by Tal-Hason** [comment]
Hi , yes i had an older release but i have uninstalled it before installing the new version

**Comment by gbraad** [comment]
this is a false positive. since we will eventually move to using Podman Desktop this will be a wontfix.
You might be able to inform Sentinel One to update their heuristics.

**Comment by anjannath** [comment]
> Hi , yes i had an older release but i have uninstalled it before installing the new version

Okay, in that case the `CodeReady Containers.lnk` file should've been removed during the uninstall and the new version upon installation shouldn't create this file again, will try to reproduce locally, thanks for creating the issue.

Or if you are willing to help out, can you please verify that during uninstall the shortcut is removed:
- uninstall the current CRC installation (2.0.1)
- install the previous version (1.39.0)
- uninstall it and see if the shortcut `CodeReady Containers.lnk` is also removed or not

**Comment by Tal-Hason** [comment]
after i uninstalled the old version non of the links and tray-icon left in the menus, i checked it and rebooted the PC to clean the cache, also i needed to remove all the old files from the c:\users\{username}\.crc\bin & c:\users\{username}\.crc\cache for the new version will be installed properly.

**Comment by Tal-Hason** [comment]
Also i still have the "CodeReady Containers.lnk" in the start menu
![Screenshot 2022-04-18 124919](https://user-images.githubusercontent.com/97454240/163791288-aa9c041a-a4ac-4ca6-af84-ce173d674e7a.png)
![Screenshot 2022-04-18 125047](https://user-images.githubusercontent.com/97454240/163791293-28af4950-c7f9-456c-b8b7-2ad40bbd069c.png)

**Comment by gbraad** [comment]
Unable to repro.

https://www.virustotal.com/gui/file-analysis/N2U0ZjQ3ZTI3YTQ4ZGQ5NDA4ZTA4YjE4YTliYjk1MWE6MTY1MDMzNDIzMA==
